config/database.yml

```diff
@@ -61,6 +61,7 @@ development:
 test:
   <<: *default
   database: ventures-be_test
+  username: <%= ENV.fetch("PGUSER") %>
```

Comment:
As we discussed, we may end up having to set an environment variable in Heroku for this also - we will see!